services/ui-src/src/utils/forms/forms.ts

```diff
@@ -86,7 +86,8 @@ export const initializeChoiceListFields = (fields: FormField[]) => {
     field?.props?.choices.forEach((choice: FieldChoice) => {
       // set choice value to choice label string
       choice.value = choice.label;
-      choice.id = choice.name;
+      choice.name = choice.id;
+      choice.id = `${field.id}-${choice.id}`;
```

Comment:
Request: Let's flip these lines so the `choice.name` and `choice.id` are always the same.